firebase-firestore/src/androidTest/java/com/google/firebase/firestore/QueryTest.java

```diff
@@ -1134,4 +1134,167 @@ public void testOrQueriesWithArrayMembership() {
         "doc4",
         "doc6");
   }
+
+  @Test
+  public void multipleInOps() {
```

Comment:
Previous functions inside this file use `test` prefix in the name, would it be better to keep this style?

Reply:
Good idea. Done.